src/inmanta/env.py

```diff
@@ -248,17 +246,13 @@ def get_packages_in_working_set(cls, inmanta_modules_only: bool = False) -> dict
         :param inmanta_modules_only: Only return inmanta modules from the working set
         """
         return {
-            packaging.utils.canonicalize_name(dist_info.key): packaging.version.Version(dist_info.version)
-            for dist_info in pkg_resources.working_set
-            if not inmanta_modules_only or dist_info.key.startswith(const.MODULE_PKG_NAME_PREFIX)
+            packaging.utils.canonicalize_name(dist_info.name): packaging.version.Version(dist_info.version)
+            for dist_info in distributions()
+            if not inmanta_modules_only or dist_info.name.startswith(const.MODULE_PKG_NAME_PREFIX)
         }
 
     @classmethod
-    def rebuild_working_set(cls) -> None:
-        pkg_resources.working_set = pkg_resources.WorkingSet._build_master()
```

Comment:
I don't know if I have to keep this and what the exact standing of this class is now. 

The working set doesn't exist any more, I'm not very clear if this class should stay or not, be renamed or not,..